packages/flutter/lib/src/widgets/interactive_viewer.dart

```diff
@@ -169,8 +181,25 @@ class InteractiveViewer extends StatefulWidget {
   /// See also:
   ///  * [constrained], which has an example of creating a table that uses
   ///    alignPanAxis.
+  @Deprecated(
+    'Use panAxis instead. '
+    'This feature was deprecated after v3.3.0-0.5.pre.',
+  )
   final bool alignPanAxis;
 
+  /// When set to [PanAxis.aligned] panning is only allowed in the horizontal
```

Comment:
Add a comma: "When set to [PanAxis.aligned],"

Reply:
Added